build/int.cloudbuild.yaml

```diff
@@ -59,6 +62,28 @@ steps:
     - verify abm-gce-gpu
   name: 'gcr.io/cloud-foundation-cicd/$_DOCKER_IMAGE_DEVELOPER_TOOLS:$_DOCKER_TAG_VERSION_DEVELOPER_TOOLS'
   args: ['/bin/bash', '-c', 'source /usr/local/bin/task_helper_functions.sh && kitchen_do destroy abm-gce-gpu']
+# ----
+# Test anthos-bm-gcp-bash sample
+- id: create abm-gce-bash
+  waitFor:
+    - destroy abm-gce-gpu
+  name: 'gcr.io/cloud-foundation-cicd/$_DOCKER_IMAGE_DEVELOPER_TOOLS:$_DOCKER_TAG_VERSION_DEVELOPER_TOOLS'
+  args: ['/bin/bash', '-c', 'source /usr/local/bin/task_helper_functions.sh && kitchen_do create abm-gce-bash']
```

Comment:
You can use the cft cli `cft test run TestABMBash --stage init --verbose --test-dir test/integration`. Kitchen is for the old ruby based test. We also have a [codelab](https://codelabs.developers.google.com/cft-onboarding#7) if you are interested.

Reply:
```suggestion
  args: ['/bin/bash', '-c', 'cft test run TestABMBash --stage init --verbose --test-dir test/integration']
```